web/src/beta/components/fields/SelectField/index.stories.tsx

```diff
@@ -68,7 +68,7 @@ export const MultiSelect: Story = (args: MultiSelectProps) => {
           name="Multi Select"
           description="You can select multiple options"
           onChange={handleChange}
-          multiSelect
+          multiSelect={true}
```

Comment:
I think we can just use `multiSelect` as a short expression for `multiSelect={true}`, no?